File: Source/ARTPendingMessage.m
#import "ARTPendingMessage.h"

@implementation ARTPendingMessage

- (instancetype)initWithProtocolMessage:(ARTProtocolMessage *)msg ackCallback:(nullable ARTMessageSendCallback)ackCallback {
    self = [super initWithProtocolMessage:msg sentCallback:nil ackCallback:ackCallback];
    return self;
}

@end

File: Sources/AblySwift/ARTPendingMessage.swift
import Foundation

// swift-migration: original location ARTPendingMessage.h, line 7 and ARTPendingMessage.m, line 3
/// :nodoc:
internal class ARTPendingMessage: ARTQueuedMessage {
    
    // swift-migration: original location ARTPendingMessage.h, line 11 and ARTPendingMessage.m, line 5
    internal init(protocolMessage msg: ARTProtocolMessage, ackCallback: ARTStatusCallback?) {
        super.init(protocolMessage: msg, sentCallback: nil, ackCallback: ackCallback)
    }
}